packages/main/test/sap/ui/webcomponents/main/samples/Select.sample.html

```diff
@@ -46,11 +46,15 @@
 </head>
 <body class="sapUiBody example-wrapper">
 
-	<header>
+	<header class="component-heading">
 		<h2 class="control-header">Select</h2>
-		<div style="margin-bottom: 2rem; font-weight: 300; font-size: 1.1rem; color: #666666;">&lt;ui5-select&gt;</div>
+		<div class="component-heading-since">
+			<span><!--since_tag_marker--></span>
+			<span class="component-heading-release-status">(next release)</span>
```

Comment:
shouldn't this be part of the since tag? I was imagining we write it as "@since 0.8.0 (next release)" and then replace it before the release.